docs/core/deploying/single-file.md

```diff
@@ -11,9 +11,26 @@ Bundling all application-dependent files into a single binary provides an applic
+
+Single-file doesn't bundle native libraries by default. On Linux, we prelink the runtime into the bundle and only application native libraries are deployed to the same directory as the single-file app. On Windows, we prelink only the hosting code and both the runtime and application native libraries are deployed to the same directory as the single-file app. This is to ensure a good debugging experience, which requires native files to be excluded from the single file. There is an option to set a flag, `IncludeNativeLibrariesForSelfExtract`, to include native libraries in the single file bundle, but these files will be extracted to a temporary directory in the client machine when the single file application is run. Similarly, single-file doesn't bundle debug files (PDB files) for a better debugging experience.
+
+Managed C++ applications aren't well suited for single-file deployment and we recommend that you write applications in C# or another non-managed C++ language to be single-file compatible. There are some rare cases where a managed library takes a dependency into the runtime library, coreclr, via `pinvoke`. This will not be possible in single-file case since the runtime library is bundled with the application.
+
+The following table outlines the API behavior in a single-file application:
+
+| API                                                                                     | Notes |
+| ---------------------------------------------------------------------------------------  | ------- |
+| Assembly.Location | Returns an empty string, use AppContext.BaseDirectory |
+| Module.FullyQualifiedName  | Returns a `<Unknown>` string ((just like it does in no-single-file for memory-loaded assemblies) |
+| Assembly.GetFile  | Will throw `IOException` (just like it does in no-single-file for memory-loaded assemblies) |
+| Assembly.GetFiles  | Will throw `IOException` (just like it does in no-single-file for memory-loaded assemblies) |
+| Assembly.CodeBase  | Will throw `PlatformNotSupportedException` (this API is obsolete in .NET Core 5.0) |
+| Assembly.EscapedCodeBase  | Will throw `PlatformNotSupportedException` (this API is obsolete in .NET Core 5.0) |
+| AssemblyName.CodeBase  | Will throw `PlatformNotSupportedException` |
```

Comment:
I do not think that's correct. I believe AssemblyName returns empty string for assemblies loaded from memory.

Reply:
You are right!

Reply:
These 2 seem to be returning null